cmd/access/node_builder/access_node_builder.go

```diff
@@ -722,6 +723,15 @@ func (builder *FlowAccessNodeBuilder) BuildExecutionSyncComponents() *FlowAccess
 					if err != nil {
 						return nil, fmt.Errorf("could not load checkpoint file: %w", err)
 					}
+
+					// this is only supported when bootstrapping a new register db, otherwise the
+					// register db will throw exceptions when indexing an already indexed height.
+					if builder.forceResetIndexerHeight {
+						err = indexedBlockHeight.SetProcessedIndex(checkpointHeight)
+						if err != nil && !errors.Is(err, storage.ErrNotFound) {
+							return nil, fmt.Errorf("could not reset indexed block height: %w", err)
+						}
```

Comment:
we should print a log before doing the operation and after it's done.

Reply:
addressed in [0d10562](https://github.com/onflow/flow-go/pull/4943/commits/0d1056235c21362f3f02caa8cefcb0150a109b91)